metatron-app/metatron-discovery
# F#3744 fix grid error

### Description
1. The fixed area to the left of the grid is not fixed.
2. If there are quotation marks in the column name, the grid is broken.
3. Columns overlap when resizing the grid.

**Related Issue** : #3744 


### How Has This Been Tested?
1. Create a grid.
2. Check that the left fixed area is fixed when you cross the grid horizontally.
2. If there are quotation marks in the column name, check if the grid is displayed normally.
3. Check that the columns are displayed without overlapping when resizing the grid.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
